src/NUnitFramework/framework/TestContext.cs

```diff
@@ -75,6 +76,18 @@ public static TextWriter Out
             get { return TestExecutionContext.CurrentContext.OutWriter; }
         }
 
+#if !NETCF && !SILVERLIGHT && !PORTABLE
+        /// <summary>
+        /// Gets a TextWriter that will send output directly to Console.Error
+        /// </summary>
+        public static TextWriter Error = new EventListenerTextWriter("Error", Console.Error);
+
+        /// <summary>
+        /// Gets a TextWriter for use in displaying immediate progress messages
+        /// </summary>
+        public static readonly TextWriter Progress = new EventListenerTextWriter("Progress", Console.Error);
```

Comment:
Exactly... sending it to Console.Error is an implementation detail - a convenience at this point. All immediate output goes to Console.Error and becomes a TestOutput event. The Stream name of the event is actually what indicates where the output is going and the runner client can do whatever it wants. For example, a GUI might put progress noticess into some special panel. All the console runner does is color the Error stream red.